outport/outport_test.go

```diff
@@ -0,0 +1,175 @@
+package outport
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/statistics"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/outport/mock"
+	"github.com/ElrondNetwork/elrond-go/outport/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewOutport(t *testing.T) {
+	t.Parallel()
+
+	outportHandler := NewOutport()
+
+	require.False(t, outportHandler.IsInterfaceNil())
+	require.False(t, outportHandler.HasDrivers())
+}
+
+func TestOutport_SaveAccounts(t *testing.T) {
+	t.Parallel()
+
+	called1 := false
+	called2 := false
+	driver1 := &mock.DriverStub{
+		SaveAccountsCalled: func(acc []state.UserAccountHandler) {
+			called1 = true
+		},
+	}
+	driver2 := &mock.DriverStub{
+		SaveAccountsCalled: func(acc []state.UserAccountHandler) {
+			called2 = true
+		},
+	}
+	outportHandler := NewOutport()
+	_ = outportHandler.SubscribeDriver(driver1)
+	_ = outportHandler.SubscribeDriver(driver2)
+
+	outportHandler.SaveAccounts([]state.UserAccountHandler{})
+	require.True(t, called1)
+	require.True(t, called2)
+}
+
+func TestOutport_SaveBlock(t *testing.T) {
+	t.Parallel()
+
+	called1 := false
```

Comment:
why all these are called `called1` and `driver1` ? (valid for all single occurrences)

Reply:
renamed.